tests/ssl-opt.sh

```diff
@@ -13873,6 +13873,16 @@ run_test    "TLS 1.2 ClientHello indicating support for deflate compression meth
             -s "dumping .client hello, compression. (2 bytes)"
 
 # Handshake defragmentation testing
+requires_openssl_3_x
+requires_protocol_version tls13
+requires_certificate_authentication
+run_test    "Handshake defragmentation on client (Disabled)" \
+            "$O_NEXT_SRV" \
+            "$P_CLI debug_level=4 " \
+            0 \
+            -C "reassembled record" \
+            -C "handshake fragment:" \
```

Comment:
I believe this should be `-c`.

Reply:
Apparently the presence of this text really depends on configuration, so I am inclined to remove it.

I could try to be more strict using something like `handshake fragment: 0 \\.\\. [0-9] of [0-9]\\+ msglen [0-9]" \` but I am not certain it will add much.

Reply:
From reading the source code I really don't think it depends on the configuration (it's specific to TLS as opposed to DTLS, but TLS is always compiled in, and those tests are all TLS anyway).

I think it's just always present and the assertion here should be `-c`.

Reply:
If I'm right, this test should never pass and in all components it will be either SKIP, or FAIL with the reason that this message was present in the client log while the test is saying it should be absent.

If I'm wrong, please show me a configuration where the message is absent.